docs/docs/installation/build-from-source/index.md

```diff
@@ -31,6 +31,10 @@ You'll need to install the following system packages before you install Mathesar
 
     _(This is required for installation only and will eventually be [relaxed](https://github.com/centerofci/mathesar/issues/2871))_
 
+- [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) 14 or newer (Verify with `node --version`)
+
+    _(This is required for installation only and will eventually be [relaxed](https://github.com/centerofci/mathesar/issues/2871))_
```

Comment:
This one specifies npm but mentions the node version.

npm version required is 6. It is checked using `npm --version`.

Reply:
Sorry, fixed it.